pkg/reconciler/route/resources/ingress.go

```diff
@@ -253,8 +264,22 @@ func makeBaseIngressPath(ns string, targets traffic.RevisionTargets) *netv1alpha
 		})
 	}
 
+	var timeoutDuration *metav1.Duration
+
+	if timeout != nil {
+		if timeout.Seconds() > float64(defaults.MaxRevisionTimeoutSeconds) {
```

Comment:
This should also be handled by validation logic in Revision webbhook, so I'd not worry about it here.

Reply:
removed